### DISCUSSION TITLE
Ability to specify driver in the drives

### DISCUSSION THREAD

**Discussion by tb205gti** [teslamate-org/teslamate#660] [Ideas]
**Is your feature request related to a problem? Please describe.**
No

No problem, but a request for a new feature

**Describe the solution you'd like**
In the drives view - be able to specify a given driver, so you at a later stage can sort on drives performed by that driver. (Gamification element, who can reach the highest efficiency)

In the drives overview, add a column with driver - be able to select the driver that performed that specific drive.

**Describe alternatives you've considered**

**Additional context**

**Comment by Dulanic** [comment]
My wife would hate me if I showed how badly she drives :)
